### DISCUSSION TITLE
Ubuntu upgrade?

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#2010] [Help]
Current situation:
Ubuntu 22.04-3 LTS
GO 3.12.1-1

Now: ubuntu wants to upgrade to 24.04-1 LTS.

Any problems to be expected?

//Jens

**Comment by rousseldenis** [comment]
> Current situation: Ubuntu 22.04-3 LTS GO 3.12.1-1
> 
> Now: ubuntu wants to upgrade to 24.04-1 LTS.
> 
> Any problems to be expected?
> 
> //Jens

Updated without problems.

I think the experience with wxwidgets 3.2 is better even if :

- reactivating already loaded window (e.g. combinations) through click on menu does not push that window to first layer.

**Reply by jason-barba** [reply]
Did you upgrade GO as well, or will my current GO version work after Ubuntu upgrade?

**Reply by rousseldenis** [reply]
I intalled the same version but with 3.2 wx widgets library(because I had issues with gtk).
